packages/flutter_tools/lib/src/localizations/gen_l10n_types.dart

```diff
@@ -657,20 +661,23 @@ class AppResourceBundleCollection {
     final RegExp filenameRE = RegExp(r'(\w+)\.arb$');
     final Map<LocaleInfo, AppResourceBundle> localeToBundle = <LocaleInfo, AppResourceBundle>{};
     final Map<String, List<LocaleInfo>> languageToLocales = <String, List<LocaleInfo>>{};
-    final List<File> files = directory.listSync().whereType<File>().toList()..sort(sortFilesByPath);
+    final List<File> files = directory
+        .listSync()
+        .whereType<File>()
+        .where((File e) => filenameRE.hasMatch(e.path))
+        .toList()
+      ..sort(sortFilesByPath);
```

Comment:
There are some spacing issues here and we should also add a comment to clarify why we need sort.
```suggestion
    // We require the list of files to be sorted so that "languageToLocales[bundle.locale.languageCode]"
    // is not null by the time we handle locales with country codes.
    final List<File> files = directory
      .listSync()
      .whereType<File>()
      .where((File e) => filenameRE.hasMatch(e.path))
      .toList()
      ..sort(sortFilesByPath);
```

Reply:
The spacing should be correct, you may run `dart format` to check the result.

Reply:
Comments are added.

Reply:
> you may run dart format to check the result.

Hi @AlexV525, thanks for contributing. The flutter/flutter repo does not use `dart format`. The formatting used in this repo is covered in detail in the [flutter style guide](https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo).

Reply:
> The spacing should be correct, you may run `dart format` to check the result.

As @Piinks said, flutter doesn't use `dart format` and typically we have no instance of extra four space indents following a line. Two space indenting each line after the `files` variable declaration is what we usually do.

Reply:
For the record, these are what I've found in the repo:
https://github.com/flutter/flutter/blob/f8afcd5aa01b3ae6c55cb6e4c9fa4171e27a92f6/packages/flutter_tools/lib/src/web/compile.dart#L237-L240
https://github.com/flutter/flutter/blob/f8afcd5aa01b3ae6c55cb6e4c9fa4171e27a92f6/packages/flutter_tools/lib/src/base/analyze_size.dart#L326-L327

Our style guide wasn't specific about how cascade notations should indent after instance callers. For me, regardless of `dart format`, it makes me quickly identify what variable the cascade is targeting.

Though I don't have a strong opinion about this, but if we suggest this, we should put it into the style guide too.

Updated.